galvan/src/main/java/org/terracotta/testing/master/ServerProcess.java

```diff
@@ -251,7 +251,7 @@ private SimpleEventingStream buildEventingStream(VerboseOutputStream stdout) {
     Map<String, String> eventMap = new HashMap<String, String>();
     eventMap.put("PID is", pidEventName);
     eventMap.put("Terracotta Server instance has started up as ACTIVE node", activeReadyName);
-    eventMap.put("console - Moved to State[ PASSIVE-STANDBY ]", passiveReadyName);
+    eventMap.put("Moved to State[ PASSIVE-STANDBY ]", passiveReadyName);
```

Comment:
are you sure this is specific to the targeted server?  I seem to remember logging statements in peer servers having this message as well when servers changed to passive standby